tests/unit/Makefile.am

```diff
@@ -13,7 +13,8 @@ tests_unit_TESTS			= \
 	tests/unit/test_hostid		   \
 	tests/unit/test_zone		   \
 	tests/unit/test_pathutils	   \
-	tests/unit/test_logwriter
+	tests/unit/test_logwriter	\
+	tests/unit/test_stats_query
 
```

Comment:
Please don't add more tests to tests/unit, rather use a more specific directory. test_stats_query should rather go to lib/stats/tests probably.